### ISSUE TITLE
Following readme instructions results in Exception.

### ISSUE THREAD

**Issue by marekrost** [nette-examples/user-authentication#4]
Following the instructions from readme file and testing the application with `php -S localhost:8000 www/index.php` will make the browser redirect to `http://localhost:8000/admin.sign/in` and result in the following exception: 

```
Nette\Application\BadRequestException #404
Cannot load presenter 'In', class 'App\Module\Presenters\InPresenter' was not found.

Caused by:
Nette\Application\InvalidPresenterException
Cannot load presenter 'In', class 'App\Module\Presenters\InPresenter' was not found.
```
PHP version used is 7.4.26.

**Comment by dg** [comment]
Related to #3

**Comment by EriiYenn** [comment]
Could I ask what is the solution, please? I followed the instructions and stopped at the same point. I read through the issues linked but seems it was not fixed. The URL still produces the malformed state of `http://localhost:8000/admin.sign/in?backlink=rk5bj` including the dot.

All I did was install your repo, allowed SQLite in PHP extensions, and tried to run the app. Is there any recent solution?

**Comment by eydun** [comment]
If you are on windows, you can use a package like Laragon.org instead of php's built-in webserver.

**Comment by EriiYenn** [comment]
> If you are on windows, you can use a package like Laragon.org instead of php's built-in webserver.

Thank you @eydun for the idea! I did my own research, but using XAMPP or Laragon ends up in a blank white screen. I also tried creating a MySQL database in the Database Manager in both XAMPP and Laragon but with no success.
When it comes to PHP I always worked with vanilla PHP, not frameworks, so this is absolutely new for me.

I tried connecting to `localhost/user-authentication-master/www/index.php` and that produced the white screen. Do I do something wrong?

**Comment by eydun** [comment]
You could try to look into C:\laragon\bin\apache\httpd-2.xxx\logs to see if the log-files show some errors.

**Comment by EriiYenn** [comment]
There is indeed one error. Error log included below happens on each manual Apache start and loading the page, NOT Apache reloads, in Laragon. I played with SSL in Laragon to allow port 443 when having problems with certificates, which ended up successfully, but the error stayed the same even when using normal HTTP. Outside of this error, nothing else was logged.

> [Sun Mar 20 22:16:24.312777 2022] [mpm_winnt:crit] [pid 20788:tid 620] AH02538: Child: Parent process exited abruptly. Child process is ending
[Sun Mar 20 22:16:26.324770 2022] [mpm_winnt:notice] [pid 20788:tid 620] AH00364: Child: All worker threads have exited.
[Sun Mar 20 22:16:27.450710 2022] [core:warn] [pid 19720:tid 544] AH00098: pid file C:/laragon/bin/apache/httpd-2.4.47-win64-VS16/logs/httpd.pid overwritten -- Unclean shutdown of previous Apache run?
[Sun Mar 20 22:16:27.486689 2022] [mpm_winnt:notice] [pid 19720:tid 544] AH00455: Apache/2.4.47 (Win64) OpenSSL/1.1.1k PHP/7.4.19 configured -- resuming normal operations
[Sun Mar 20 22:16:27.486689 2022] [mpm_winnt:notice] [pid 19720:tid 544] AH00456: Apache Lounge VS16 Server built: Apr 24 2021 11:08:47
[Sun Mar 20 22:16:27.486689 2022] [core:notice] [pid 19720:tid 544] AH00094: Command line: 'C:\\laragon\\bin\\apache\\httpd-2.4.47-win64-VS16\\bin\\httpd -d C:/laragon/bin/apache/httpd-2.4.47-win64-VS16'
[Sun Mar 20 22:16:27.506685 2022] [mpm_winnt:notice] [pid 19720:tid 544] AH00418: Parent: Created child process 22192
[Sun Mar 20 22:16:28.142312 2022] [mpm_winnt:notice] [pid 22192:tid 672] AH00354: Child: Starting 64 worker threads.

I did my research on it, but nothing helped.
[first link](https://stackoverflow.com/questions/56501601/apache-error-ah02538-child-parent-process-exited-abruptly-child-process-is-en)
[second link](https://stackoverflow.com/questions/1138269/apache-error-notice-parent-child-process-exited-with-status-3221225477-res)

**TL;DR** - Still white screen, but the Nette favicon loaded :smile:. HTTP error 500 on any request.
I might consider moving onto Django as I am much more familiar with it and it works flawlessly. 🥲 

Thanks for the advice! If you happen to know the solution or provide some screenshots or files for correct configuration, I will appreciate it :heart:

**Comment by dg** [comment]
Closed by nette/http 3.1.6
